pkg/apis/pipeline/v1beta1/pipelinerun_conversion_test.go

```diff
@@ -724,16 +702,92 @@ func TestPipelineRunConversionEmbeddedStatusConvertFrom(t *testing.T) {
 				},
 			},
 		},
+		embeddedStatus: config.BothEmbeddedStatus,
+	}, {
+		name: "minimal",
+		in: &v1.PipelineRun{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "foo",
+				Namespace: "bar",
+			},
+			Spec: v1.PipelineRunSpec{
+				PipelineRef: &v1.PipelineRef{
+					Name: "test",
+				},
+			},
+			Status: v1.PipelineRunStatus{
+				PipelineRunStatusFields: v1.PipelineRunStatusFields{
+					ChildReferences: childRefs,
+				},
+			},
+		},
+		want: &v1beta1.PipelineRun{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "foo",
+				Namespace: "bar",
+			},
+			Spec: v1beta1.PipelineRunSpec{
+				PipelineRef: &v1beta1.PipelineRef{
+					Name: "test",
+				},
+			},
+			Status: v1beta1.PipelineRunStatus{
+				PipelineRunStatusFields: v1beta1.PipelineRunStatusFields{
+					ChildReferences: append(childRefTaskRuns, childRefRuns...),
+				},
+			},
+		},
+		embeddedStatus: config.MinimalEmbeddedStatus,
+	}, {
+		name: "full",
+		in: &v1.PipelineRun{
+			ObjectMeta: metav1.ObjectMeta{
+				Name:      "foo",
+				Namespace: "bar",
+				Annotations: map[string]string{
```

Comment:
wonder if it makes sense to make this map as a var if it is used in different cases?

Reply:
Sorry can you elaborate more about what it means by making it into a var? From my understanding, the annotation is a field in ObjectMeta https://kubernetes.io/docs/concepts/overview/working-with-objects/annotations/ ?

Reply:
done